packages/@aws-cdk/aws-batch/lib/job-definition.ts

```diff
@@ -112,6 +112,13 @@ export interface JobDefinitionContainer {
    */
   readonly environment?: { [key: string]: string };
 
+  /**
+   * The environment variables from secrets manager or ssm parameter store
```

Comment:
`ecs.Secret` will not support SSM parameter store though.

Don't you want `core.SecretValue` ?

Reply:
Well I guess not if you need the ARN. But in any case, the SSM comment here is inaccurate.

Reply:
To my understanding, `ecs.Secret` supports SSM parameter store through `fromSsmParameter` method.
Is it correct?

Reply:
Sorry, I was thinking of `ssm.Secret`.